### PR TITLE
feat: report checksummer userop.sender & markup on gas price

### PR DESCRIPTION
**Pull request by 0xSulpiride**
## Description

- p2p: report checksummed userop.sender #111 
- add markup on the gas price reported by eth_estimateUserOp

## Types of changes
What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Further comments (optional)

-

### CODE REVIEW

File: packages/executor/src/services/BundlingService.ts [hunk 1]
```diff
@@ -148,8 +148,7 @@ export class BundlingService {
           // eslint-disable-next-line @typescript-eslint/no-unused-vars
           const { gasLimit, ...txWithoutGasLimit } = tx;
           // some chains, like Bifrost, don't allow setting gasLimit in estimateGas
-          const estimatedGasLimit = await wallet.estimateGas(txWithoutGasLimit);
-          tx.gasLimit = estimatedGasLimit;
+          await wallet.estimateGas(txWithoutGasLimit);
```
Comment (ch4r10t33r) [inline packages/executor/src/services/BundlingService.ts:151]: Why have we removed the gasLimit check here? I thought that was affecting Gnosis/Bifrost chain?

File: packages/executor/src/modules/eth.ts [hunk 2]
```diff
@@ -190,21 +192,11 @@ export class Eth {
...
+    const gasFee = await this.skandhaModule.getGasPrice();
 
     if (this.pvgEstimator) {
-      userOpComplemented.maxFeePerGas =
-        gasFee.maxFeePerGas ?? gasFee.gasPrice ?? 1;
-      userOpComplemented.maxPriorityFeePerGas =
-        gasFee.maxPriorityFeePerGas ?? gasFee.gasPrice ?? 1;
+      userOpComplemented.maxFeePerGas = gasFee.maxFeePerGas;
+      userOpComplemented.maxPriorityFeePerGas = gasFee.maxPriorityFeePerGas;
       preVerificationGas = await this.pvgEstimator(
```
Comment (ch4r10t33r) [inline packages/executor/src/modules/eth.ts:200]: Should we consider adding a markup here?

File: packages/executor/src/services/BundlingService.ts (hunk 1 above)
Comment (0xSulpiride) [reply packages/executor/src/services/BundlingService.ts:151]: the estimated gasLimit is too low. There's a check in the EntryPoint contract that requires setting a higher gas limit. The purpose of calling `wallet.estimateGas` here is too make sure that the transaction does not revert

File: packages/executor/src/modules/eth.ts (hunk 2 above)
Comment (0xSulpiride) [reply packages/executor/src/modules/eth.ts:200]: it's already added in `skandhaModule.getGasPrice`

**Review by ch4r10t33r — APPROVED** [review]
LGTM

**Pull request merged**